### PR TITLE
py-postgresql: update to 1.3.0

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Drop python 3.7, use newer ones.

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: python/py-postgresql/Portfile [hunk 1]
```diff
@@ -5,31 +5,29 @@ PortGroup           python 1.0
...
 license             BSD
 maintainers         nomaintainer
 description         Python interface to PostgreSQL
-long_description \
-   py-postgresql is a project dedicated to improving the Python interfaces \
-   to PostgreSQL. It strives to provide substantial convenience to Python \
-   programmers using PostgreSQL.  At its core, py-postgresql provides a \
-   PG-API, postgresql.api, and DB-API 2.0 interface for using a PostgreSQL \
-   database.
+long_description    ${name} is a project dedicated to improving the Python interfaces \
```
Comment (pmetzger) [inline python/py-postgresql/Portfile:14]: Is this still narrower than 80 columns?

Comment (barracuda156) [reply python/py-postgresql/Portfile:14]: I have no idea but I can make it narrower than it is.

Comment (pmetzger) [reply python/py-postgresql/Portfile:14]: Thanks. Please keep portfiles (except for the modeline) to 80 columns.

Comment (barracuda156) [reply python/py-postgresql/Portfile:14]: I made description narrower now.

Comment (pmetzger) [reply python/py-postgresql/Portfile:14]: But it's still much wider than 80 columns. I'm not sure what the problem was with the old version of this, which was under 80 columns.

Comment (barracuda156) [reply python/py-postgresql/Portfile]: I believe, description is not squeezed into 80 character-width.

**Comment by pmetzger** [comment]
Thanks for fixing that!

**Pull request merged**